Makefile

```diff
@@ -1,148 +1,99 @@
-#
 # Copyright 2015 Panagiotis Papadomitsos. All Rights Reserved.
```

Comment:
^ Should Miniclip's own copyright notice be placed below?

Otherwise it sort of implies that our fellow Panagiotis here made all the stuff we did, down below - I don't know if that might present a legal problem.

But IANAL.

Reply:
We've not done it before, but probably can. I'll put this in the files I changed.

Reply:
801d0f4

Reply:
👍